lib/profiler/get_tick.h

```diff
@@ -67,6 +67,15 @@ get_tick(void)
     return regs;
 }
 
+#elif defined(HAVE_CLOCK_GETTIME_NSEC_NP)
+#include <time.h>
+
+static __inline hrtime_t
+get_tick(void)
+{
+    return clock_gettime_nsec_np (CLOCK_UPTIME_RAW);
```

Comment:
If possible (I have not checked!), let's use a more standard constant:

```suggestion
    return clock_gettime_nsec_np (CLOCK_MONOTONIC_RAW);
```

This change request was inspired, in part, by a similar LVM [change](https://reviews.llvm.org/rG8bec8927134f116eb11ddea9db78b1a6241884c9).